packages/core/src/operations.ts

```diff
@@ -183,4 +183,21 @@ export class TxOperations implements Omit<Client, 'notify'> {
     }
     return this.removeDoc(doc._class, doc.space, doc._id)
   }
+
+  async updateWithAttachments<T extends Doc>(doc: T, update: DocumentUpdate<T>, retrieve?: boolean): Promise<void> {
```

Comment:
Please move to server/storage/ as trigger for TxUpdateDoc, similar to processRemove.

Reply:
Done